service/src/test/java/uk/gov/hmcts/reform/fpl/service/SentDocumentHistoryServiceTest.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.fpl.model.DocumentSentToParty;
+import uk.gov.hmcts.reform.fpl.model.DocumentsSentToParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.testDocument;
+
+public class SentDocumentHistoryServiceTest {
+
+    private static final String PARTY_1 = "John Smith";
+    private static final String PARTY_2 = "Alex Green";
+
+    SentDocumentHistoryService sentDocumentHistoryService = new SentDocumentHistoryService();
```

Comment:
would another test class be extending `SentDocumentHistoryServiceTest` which would need `sentDocumentHistoryService`?

Reply:
rather not, i dont think that extending test class is good idea